components/skus/renderer/skus_utils.cc

```diff
@@ -5,24 +5,37 @@
 
 #include "brave/components/skus/renderer/skus_utils.h"
 
+#include <algorithm>
+#include <string>
 #include <vector>
 
 #include "base/no_destructor.h"
 #include "third_party/blink/public/platform/web_security_origin.h"
 #include "third_party/blink/public/platform/web_url.h"
 #include "url/gurl.h"
 
-namespace skus {
+namespace {
+// NOTE: please open a security review when appending to this list.
+std::vector<GURL> safe_origins_gurl{GURL("https://account.brave.com"),
+                                    GURL("https://account.bravesoftware.com"),
+                                    GURL("https://account.brave.software")};
```

Comment:
Globals with non-trivial destructors like this one, have to be declared as `NoDestructor`. However, I think this can be simplified just to

```cxx
constexpr auto kSafeOrigins = std::to_array<std::string_view>({
  "https://account.brave.com",
  "https://account.bravesoftware.com",
  "https://account.brave.software",
})
```

And then you can wrap it on a GURL inside the `std::transform` operation.

Reply:
Fixed!